### DISCUSSION TITLE
Support for iOS Icon Variations (iOS 18 Dark and Tinted Icons)

### DISCUSSION THREAD

**Discussion by KevinvdBurg** [expo/expo#31501] [Proposals and Ideas]
Hi,

Currently, there is no way to add icon variations for iOS in Expo, similar to how we can for Android with adaptive icons. With the release of iOS 18, Apple has introduced guidelines for app icon variations, including dark and tinted variants:

[Apple Developer Documentation - Configuring Your App Icon](https://developer.apple.com/documentation/xcode/configuring-your-app-icon/#Add-dark-and-tinted-icon-variants-to-iOS-and-iPadOS)

**Proposal:**

Add support in Expo for specifying iOS icon variations using a configuration similar to Android's `adaptiveIcons`. 

Example configuration:

```json
{
  "ios": {
    "icon": {
      "any": "./assets/images/icon.png",
      "dark": "./assets/images/icon-dark.png",
      "tinted": "./assets/images/icon-tinted.png"
    }
  }
}
```

**Comment by RyanTG** [comment]
Being discussed here, as well https://github.com/expo/expo/pull/30247

**Reply by fobos531** [reply]
Helo @KevinvdBurg - the linked PR has been merged and should become available to users soon.
